### PR TITLE
API: Improve dependencies report

### PR DESCRIPTION
**Pull request by nkeim**
This corrects some of the package names in `diag.dependencies()` and adds trackpy as a "dependency" to aid in diagnostics. It also puts `dependencies()` into the root namespace, so it will eventually be easier to ask people to run it.

This was originally part of #566 but I'm making it a separate PR.

### CODE REVIEW

**Comment by nkeim** [comment]
I changed my mind on this: adding `trackpy.dependencies()` means extra short-term confusion for a small long-term benefit. So it is staying as `trackpy.diag.dependencies()`. In the meantime I realized that it should also report the `sklearn` version.

This should be ready to merge.

File: trackpy/diag.py [hunk 1]
```diff
@@ -30,8 +31,12 @@ def dependencies():
         optional dependency is not installed, None
     """
     packages = ['six', 'numpy', 'scipy', 'matplotlib', 'pandas',
-                'scikit-image', 'pyyaml', 'pytables', 'numba', 'pyfftw']
+                'skimage', 'sklearn', 'pyyaml', 'tables', 'numba', 'pyfftw']
```
Comment (caspervdw) [inline trackpy/diag.py]: I think pyfftw can be dropped now that we moved pyfftw to the `legacy_bandpass` function.

Also, the exception for pyfftw below can be cleaned up then

Comment (nkeim) [reply trackpy/diag.py]: Good call! I dropped pyfftw and added pims, since that seems to come up often when troubleshooting trackpy.

**Comment by nkeim** [comment]
Thanks, @caspervdw !

**Pull request merged**